package.json

```diff
@@ -20,16 +20,18 @@
     "pretty-check": "prettier --check './**/*.js'",
     "test": "truffle test",
     "build": "yarn compile && yarn networks-inject",
+    "copy-artifacts": "cp ./node_modules/@gnosis.pm/dex-contracts/build/contracts/BatchExchange.json ./build/contracts",
     "compile": "truffle compile",
     "testnet": "ganache-cli --gasLimit 8e6",
     "migrate": "truffle migrate",
     "verify": "truffle run verify",
     "flatten": "export FLATTENED_DIR=\"./build/flattenedContracts\" && mkdir -p $FLATTENED_DIR && npx sol-merger \"contracts/*.sol\" $FLATTENED_DIR && echo \"Flattened contracts stored in $FLATTENED_DIR\" && unset FLATTENED_DIR",
-    "networks-inject": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
-    "networks-extract": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js"
+    "networks-inject": "CONF_FILE=$(pwd)'/migrations/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js  && CONF_FILE=$(pwd)'/migrations/migration_conf_dex-contracts.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
```

Comment:
I agree! But this requires rewriting parts of @gnosis.pm/util-contracts, which is out of scope for this PR. Or do you see an easy solution for that?

Reply:
Good point, I don't see how it can be done with just Javascript without touching the dependency directly. I tried to come up with another way to remove the redundant config in PR #419, but I'm not sure how much of an improvement it would be. Feel free to merge or reject as you prefer.